pkgs/by-name/mo/mozlz4/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mozlz4";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jusw85";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-faoFvDvqCr0EPsqlQvHZpd74b5pzYhtkJ6gXebwEm/w=";
```

```suggestion
    repo = "mozlz4";
    tag = "v${version}";
    hash = "sha256-faoFvDvqCr0EPsqlQvHZpd74b5pzYhtkJ6gXebwEm/w=";
```
Use of pname like this is an antipattern and bad practice as it breaks overrides.